.maintain/playbooks/client-upgrade.yml

```diff
@@ -7,10 +7,6 @@
           become: true
           command: supervisorctl stop composable
     
-        # - name: Remove Previous Version #ToDO: add --continue-on-error
-        #   become: yes
-        #   command: rm /usr/bin/composable
-    
         - name: Download composable binary {{ release_version }}
           become: yes
           get_url:
```

Comment:
Should this be blank `get_url` ?

Reply:
The url is not blank...its on the next line